armory/datasets/preprocessing.py

```diff
@@ -0,0 +1,101 @@
+"""
+Standard preprocessing for different datasets
+"""
+
+
+import tensorflow as tf
+
+
+REGISTERED_PREPROCESSORS = {}
+
+
+def register(function, name=None):
+    if name is None:
+        name = function.__name__
+    global REGISTERED_PREPROCESSORS
+    REGISTERED_PREPROCESSORS[name] = function
+    return function
+
+
+def list_registered():
+    return list(REGISTERED_PREPROCESSORS)
+
+
+def get(name, version=None):
+    if version is not None:
+        raise NotImplementedError("version is not None for preprocessing to_canon")
```

Comment:
why does `actual_loader()` pass along a `version` kwarg to `preprocessing.get()` if the latter shouldn't receive anything other than `None` for this? Is the idea that eventually `REGISTERED_PREPROCESSORS` will have keys for different versions of the same dataset?

Reply:
Removed version from that.